problema1

```diff
@@ -0,0 +1,15 @@
+
+#  def even_sum0(a)
+#    suma = 0
+#    a.each { |i|
+#      if i%2 == 0 then suma += i end
+#    }
+#    suma
+#  end
+
+  def even_sum(a)
+    a.find_all { |i| i%2 == 0 }.inject(0){|sum,x| sum + x }
```

Comment:
(array#select)[http://ruby-doc.org//core-2.2.0/Array.html#method-i-select]

`even?` is so much nicer